api/swagger-spec/oapi-v1.json

```diff
@@ -13819,6 +13819,11 @@
      "resources": {
       "$ref": "v1.ResourceRequirements",
       "description": "the desired compute resources the build should have"
+     },
+     "maxDuration": {
```

Comment:
activeDeadlineSeconds is not a good name.  it implies a deadline for something to become active.  it does not imply a maximum amount of time something is allowed to be active.  Not to mention "deadline" implies a specific point in time, not a length of time.  activeDurationSeconds might have made sense.

so my justification is i want the buildconfig api to actually make sense when people look at the field, they will intuitively understand what the field does.

And i don't see any reason this has to match the underlying api we are using to implement it.  We could choose to implement build duration limits with any manner of mechanisms.  That we happened to choose to implement it by leveraging an underlying api that's not being exposed to the user here is irrelevant to how we name this field.

if it has to end in DeadlineSeconds then i want it named "buildDurationDeadlineSeconds"